database/seed.js

```diff
@@ -22,28 +22,44 @@ var itemSchema = mongoose.Schema({
 
 var itemList = mongoose.model('itemList', itemSchema);
 
-var createSummary = function() {
-  for (var itemId = 1; itemId <= 100; itemId++) {
-    var newItem = new itemList({
+itemList.deleteMany({}, () => {
+  console.log('Database dropped.');
+});
+
+let createSummary = function() {
+  for (let itemId = 1; itemId <= 100; itemId ++) {
+    let newItem = new itemList({
       productId: itemId, 
       shop: 'My Shop',
       location: faker.address.city() + ', ' + faker.address.zipCode(),
       description: faker.lorem.sentence(), 
-      price: faker.commerce.price(),
-      image: faker.image.fashion()
+      image: faker.image.fashion(),
+      price: faker.commerce.price()
     });
-    newItem.save((err, itemId) => {
+    newItem.save((err) => {
```

Comment:
just a comment: single argument arrow functions don't need parenthesis.  The following two lines are equivalent:
1) `(err) => {}`
2) `err => {}`